unittests/integration/get_state_tester.cpp

```diff
@@ -165,3 +165,28 @@ CUDAQ_TEST(GetStateTester, checkOverlapFromHostVector) {
   EXPECT_NEAR(1.0, state.overlap(hostState).real(), 1e-3);
 }
 #endif
+
+#ifdef CUDAQ_SIMULATION_SCALAR_FP64
+using SimScalarType = double;
```

Comment:
Can't you use `cudaq::real` ?

Reply:
> Can't you use `cudaq::real` ?

Ah, thanks. I didn't see that. Updated in 1e04dd696.